static/app/views/profiling/landing/profilingSlowestTransactionsPanel.tsx

```diff
@@ -0,0 +1,226 @@
+
+import {Button} from 'sentry/components/button';
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {Panel} from 'sentry/components/panels';
+import PerformanceDuration from 'sentry/components/performanceDuration';
+import {Flex} from 'sentry/components/profiling/flex';
+import {
+  FunctionsMiniGrid,
+  FunctionsMiniGridEmptyState,
+  FunctionsMiniGridLoading,
+} from 'sentry/components/profiling/functionsMiniGrid';
+import {TextTruncateOverflow} from 'sentry/components/profiling/textTruncateOverflow';
+import {IconChevron} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Organization, Project} from 'sentry/types';
+import {
+  EventsResultsDataRow,
+  useProfileEvents,
+} from 'sentry/utils/profiling/hooks/useProfileEvents';
+import {useProfilingTransactionQuickSummary} from 'sentry/utils/profiling/hooks/useProfilingTransactionQuickSummary';
+import {generateProfileSummaryRoute} from 'sentry/utils/profiling/routes';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+
+const fields = ['transaction', 'project.id', 'last_seen()', 'p95()', 'count()'] as const;
+type SlowestTransactionsFields = typeof fields[number];
+
+export function ProfilingSlowestTransactionsPanel() {
+  const profilingTransactionsQuery = useProfileEvents({
+    fields,
+    sort: {
+      key: 'p95()',
+      order: 'desc',
+    },
+    limit: 3,
+    query: 'count():>3',
+    referrer: 'api.profiling.landing-slowest-transaction-panel',
+  });
+
+  const [openPanel, setOpenPanel] = useState<null | string>(null);
+
+  const profilingTransactions = useMemo(
+    () => profilingTransactionsQuery.data?.[0].data ?? [],
+    [profilingTransactionsQuery.data]
+  );
+
+  const transactionNames = useMemo(
+    () => profilingTransactions.map(txn => txn.transaction),
+    [profilingTransactions]
+  );
+
+  if (transactionNames.length > 0 && !transactionNames.includes(openPanel)) {
+    const firstTransaction = transactionNames[0];
+    setOpenPanel(firstTransaction as string);
+  }
+
+  const {isLoading} = profilingTransactionsQuery;
+  const hasProfilingTransactions =
+    !isLoading && profilingTransactions && profilingTransactions.length > 0;
+
+  return (
+    <FlexPanel>
+      <Flex column h="100%">
+        <Flex column p={space(1.5)}>
+          <PanelHeading>{t('Slowest Transactions')}</PanelHeading>
+          <PanelSubheading>
+            {t('Slowest transactions that could use some optimization.')}
+          </PanelSubheading>
+        </Flex>
+
+        {(isLoading || !hasProfilingTransactions) && (
+          <Flex column align="center" justify="center" h="100%">
+            {isLoading ? (
+              <LoadingIndicator />
+            ) : (
+              !hasProfilingTransactions && (
+                <Flex.Item>{t('No results found for your query')}</Flex.Item>
```

Comment:
self: add a tooltip or some type of content explaining there is a `count():>3` limit on this